lib/smart_answer/flow.rb

```diff
@@ -16,18 +16,18 @@ def initialize(&block)
       instance_eval(&block) if block_given?
     end
 
+    def append(flow)
+      flow.nodes.each do |node|
+        node.flow = self
```

Comment:
> Can you be any more specific about your concern?

@floehopper: Without really thinking about it; I wondered whether there might be a problem with `FlowRegistry` caching and setting `node.flow = self`. I wondered whether it might be possible for the inner flow's nodes to have the parent flow cached such that it caused problems when the inner flow was used by two separate parent flows.

I'm not sure that makes much/any sense but I'm not too worried as your explanation sounds good to me :-)